Presentation.Web/app/components/org/user/org-user-delete.modal.view.html

```diff
@@ -104,6 +106,10 @@ <h5 ng-if="ctrl.noRights">Brugeren har ikke nogen tildelte roller i denne organi
     </table>
 </div>
 <div class="modal-footer">
+    <button type="button" class="btn btn-default pull-left" data-ng-click="ctrl.selectOrDeselectAll()" data-disabled-on-busy>
+        <ng-container ng-if="!ctrl.allSelections">Vælg alle</ng-container>
+        <ng-container ng-if="ctrl.allSelections">Fravælg alle</ng-container>
```

Comment:
why not create different buttons with an explicit call "selectAll" or "deselectAll" in stead of one with different text and a combined function.. it just makes the function more complicated.